### PR TITLE
Nitpicks about consistencies :meowglaresip:

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: faq.yaml [hunk 1]
```diff
@@ -5,26 +5,29 @@ Introduction:
...
     How do I add Rythm to my server?: |
         <br/> You can add Rythm to your Discord server by clicking on the following link:
+        <br/>
         <br/> <a href="https://rythmbot.co/invite">https://rythmbot.co/invite</a>
-        <br/> <b>Note:</b> You must have a role with “Administrator” or “Manage Server” permissions to be able to add Rythm to a Discord Server. If you have either of these permissions you should be able to change the name of your server.
-        <br/> Also make sure you’re <b><a href="https://discordapp.com/login">logged in</a></b> to the right account in the web browser when adding Rythm, see <a href="https://discordapp.com/login">https://discordapp.com/login</a>.
+        <br/>
+        <br/> <b>Note:</b> You must have a role with “Administrator” or “Manage Server” permission to be able to add Rythm to a Discord Server. If you have either of these permissions you should be able to change the name of your server.
```
Comment (Repulser) [inline faq.yaml]: the "Manage Server" permission?

File: faq.yaml [hunk 2]
```diff
@@ -5,26 +5,29 @@ Introduction:
...
-        <br/> <b>Note:</b> For an invite to Rythm's Discord server, use the following link: <a href="https://rythmbot.co/support">https://rythmbot.co/support</a>
+        <br/> <b>Note:</b> For an invite to Rythm's Discord server, use the following link:
+        <br/>
+        <br/> <a href="https://rythmbot.co/support">https://rythmbot.co/support</a>
     How can Rythm be in so many guilds at once?: |
         <br/> Rythm uses a system called "sharding".
         <br/>
-        <br/> For more information by discord's developers <a href="https://discordapp.com/developers/docs/topics/gateway#sharding">click here (https://discordapp.com/developers/docs/topics/gateway)</a>.
-        <br/> (Scroll down until you reach a section titled <code>sharding</code>)
+        <br/> For more information on how sharding works <a href="https://discordapp.com/developers/docs/topics/gateway#sharding">click here (https://discordapp.com/developers/docs/topics/gateway#sharding)</a> to view Discord's developers documentation of it.
```
Comment (Repulser) [inline faq.yaml]: No need for developers

File: faq.yaml [hunk 3]
```diff
@@ -196,7 +202,7 @@ Roles/Permissions:
         <br/> <b>•</b> Don’t have access to control the music in any way.
         <br/>
         <br/>
-        <br/> <u>Users with a <b>DJ role</b> / <b>Manage Channel</b> permissions</u>:
+        <br/> <u>Users with a <b>DJ role</b> / <b>Manage Channels</b> permission</u>:
```
Comment (Repulser) [inline faq.yaml:197]: the permission

File: faq.yaml [hunk 4]
```diff
@@ -205,20 +211,20 @@ Roles/Permissions:
         <br/> with no special permissions assigned and it will still access these features.
         <br/>
         <br/>
-        <br/> <u>Users with <b>Manage Server</b> / <b>Administrator</b> permissions</u>:
+        <br/> <u>Users with <b>Manage Server</b> / <b>Administrator</b> permission</u>:
```
Comment (Repulser) [inline faq.yaml:206]: the permission

File: faq.yaml [hunk 5]
```diff
@@ -205,20 +211,20 @@ Roles/Permissions:
...
         <br/>
-        <br/> • <b>Manage Server</b>: Users with Manage server permissions can change Rythm’s settings,
+        <br/> • <b>Manage Server</b>: Users with Manage Server permission can change Rythm’s settings,
```
Comment (Repulser) [inline faq.yaml]: the permission

File: faq.yaml [hunk 6]
```diff
@@ -205,20 +211,20 @@ Roles/Permissions:
...
         <br/> but are limited to <b>only</b> use the normal user's music commands.
         <br/>
-        <br/> • <b>Administrator</b>: Users with Administrator permissions have full access to all music commands
+        <br/> • <b>Administrator</b>: Users with Administrator permission have full access to all music commands
```
Comment (Repulser) [inline faq.yaml]: the permission

File: faq.yaml [hunk 7]
```diff
@@ -236,58 +242,57 @@ Roles/Permissions:
         <br/>
         <br/> <b>Example</b>: !settings blacklist #general
     How do I give normal users access to all music commands?: |
-        <br/> Users with <b>Manage Channels</b> permissions have access to all music commands by <b>default</b>.
-        <br/> Users with <b>Manage Server</b> permissions <b>don’t</b>.
+        <br/> Users with <b>Manage Channels</b> permission have access to all music commands by <b>default</b>.
```
Comment (Repulser) [inline faq.yaml]: the permission

File: faq.yaml [hunk 8]
```diff
@@ -236,58 +242,57 @@ Roles/Permissions:
...
+        <br/> To restrict Rythm's queue to a specific number of songs per user you must use the settings menu.
         <br/>
         <br/> To limit the number of songs per user use the following command format:
         <br/>
         <br/> <b>!settings maxusersongs <code>Number</code>/<code>disable</code></b>
         <br/>
         <br/> <b>Example</b>: !settings maxusersongs 5
         <br/>
-        <br/> Now any songs added that will put the queue over this number for a specific user will be ignored
+        <br/> Now any songs added that will put the queue over this number for a specific user will be blocked.
```
Comment (Repulser) [inline faq.yaml]: put in the queue?

File: faq.yaml [hunk 9]
```diff
@@ -236,58 +242,57 @@ Roles/Permissions:
...
+        <br/> To restrict Rythm's queue to a specific number of songs you must use the settings menu.
         <br/>
         <br/> To limit the number of songs use the following command format:
         <br/>
         <br/> <b>!settings maxqueuelength <code>Number</code></b>
         <br/>
         <br/> <b>Example</b>: !settings maxqueuelength 25
         <br/>
-        <br/> Now any songs added that will put the queue over this number will be ignored
+        <br/> Now any songs added that will put the queue over this number will be blocked.
```
Comment (Repulser) [inline faq.yaml:287]: put in the queue?

File: faq.yaml (hunk 4 above)
Comment (ghost) [reply faq.yaml:206]: I don't think "the" is needed here with how it's phrased.

File: faq.yaml [hunk 10]
```diff
@@ -51,7 +54,7 @@ Basics:
         <br/>
         <br/> A message should be returned by Rythm stating its current prefix.
     Can I play a playlist with Rythm?: |
-        <br/> Rythm can reproduce playlists from a variety of online audio sources including youtube and soundcloud.
+        <br/> Rythm can reproduce playlists from a variety of online audio sources including YouTube and SoundCloud.
```
Comment (ghost) [inline faq.yaml:57]: Would "Rythm can use playlists..." be better for here?

Comment (Repulser) [reply faq.yaml:57]: Agreed

**Pull request merged**